recipe/meta.yaml

```
@@ -171,6 +182,7 @@ outputs:
         - lit
       commands:
         - lit -h
```

```suggestion
      requires:
        - pip
      commands:
        pip check
        - lit -h
```
Because of existing https://pypi.org/project/lit/

---

done